### PR TITLE
feat(device): Device Playback 

### PR DESCRIPTION
**Pull request by AleksaBajat**
**Description**:

- Tossed out the animation controller (it was making noise, and it's not required for MVP)
- PlaybackBloc initializes the Video Controller before sending it to the UI (this was intended way of using it)
- Disposed the previous Video Controller before emitting a new one.
- requestLegacyExternalStorage enabled
- New diagnostics view when built with "DEBUG".

**Checklist**:

- [ ] Unit tests: Have you added or updated unit tests for your changes and do you need to?
- [ ] Integration tests: Have you added or updated integration tests and do you need to?
- [ ] Bechmarks: Have you added benchmarks and if so did you run them?
- [ ] Documentation: Have you update the relevant documentation to reflect the changes?

**Additional notes**:
Here you can write what ever you feel is needed and isn't covered by parts above.

### CODE REVIEW

File: launcher/lib/src/channels/bloc/playback_bloc.dart [hunk 1]
```diff
@@ -20,6 +20,7 @@ final class PlaybackPlayNext extends PlaybackEvent {}
 class PlaybackState {
   final Queue<ScheduleItem> playbackQueue;
   final VideoPlayerController? current;
+//  final VideoPlayerController? next;
```
Comment (NikolaMilosa) [inline launcher/lib/src/channels/bloc/playback_bloc.dart:23]: nit: Remove if not used

File: launcher/lib/src/channels/bloc/playback_bloc.dart [hunk 2]
```diff
@@ -73,6 +80,12 @@ class PlaybackBloc extends Bloc<PlaybackEvent, PlaybackState> {
           }
 
           playerController = VideoPlayerController.file(File(path));
+          await playerController.initialize();
+          await playerController.setLooping(false);
```
Comment (NikolaMilosa) [inline launcher/lib/src/channels/bloc/playback_bloc.dart:84]: nit(maybe?): if these two functions are always meant to be called together, wrap them in another function accepting a boolean which will be passed to setLooping

**Review by NikolaMilosa — APPROVED** [review]
Looks great! Love the focus on the MVP and removing unneeded things for now! Thanks

File: launcher/lib/src/channels/bloc/playback_bloc.dart (hunk 1 above)
Comment (AleksaBajat) [reply launcher/lib/src/channels/bloc/playback_bloc.dart]: Removed!

File: launcher/lib/src/channels/bloc/playback_bloc.dart (hunk 2 above)
Comment (AleksaBajat) [reply launcher/lib/src/channels/bloc/playback_bloc.dart]: Wrote an extension function to encapsulate this and future initialization behavior.

**Review by NikolaMilosa — APPROVED** [review]

**Pull request merged**